ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +250,73 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Operation Throughput:
+                        units: operations/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsCompleted
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesCompleted
+                                lineType: LINE
+                    Merge Rate:
+                        units: operations merged/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsMerged
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesMerged
+                                lineType: LINE
+                    Sector Throughput:
+                        units: sectors/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Read:
+                                dpName: diskstats_sectorsRead
+                                lineType: LINE
+                            Written:
+                                dpName: diskstats_sectorsWritten
+                                lineType: LINE
+                    IO Throughput:
```

Comment:
ioInProgress isn't a measure of IO Throughput, so I'd call the graph something else. Perhaps just "IO Operations in Progress" then set the units to "operations".